apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.ts

```diff
@@ -321,3 +331,55 @@ function CopyCourseInstanceForm({
     </div>
   `;
 }
+
+function CourseInstanceSharing({
+  courseInstance,
+  publicLink,
+}: {
+  courseInstance: CourseInstance;
+  publicLink: string;
+}) {
+  if (!courseInstance.share_source_publicly) {
+    return html`<p>This course instance is not being shared.</p>`;
+  }
+
+  return html`
+    <p>
+      <span class="badge color-green3 me-1">Public source</span>
+      This course instance's source is publicly shared.
+    </p>
+    <div>
+      <label for="publicLink">Public Link</label>
```

Comment:
Nit: our label capitalization is all over the place on this page. We don't have to change them here, but it'd be nice to be more consistent. I'd lean towards using sentence case.